### ISSUE TITLE
Add Total Variation (TV)

### ISSUE THREAD

**Issue by jshin1394** [google/metrax#63]

**Comment by jshin1394** [comment]
Using Keras [implementation](https://www.tensorflow.org/api_docs/python/tf/image/total_variation) as base, add the metric to metrax.

**Comment by Mrigankkh** [comment]
Hello! Can I take this up?

**Comment by jshin1394** [comment]
sure :)
